src/ribbon/gallery.cpp

```diff
@@ -36,7 +36,7 @@ wxDEFINE_EVENT(wxEVT_RIBBONGALLERY_CLICKED, wxRibbonGalleryEvent);
 wxIMPLEMENT_DYNAMIC_CLASS(wxRibbonGalleryEvent, wxCommandEvent);
 wxIMPLEMENT_CLASS(wxRibbonGallery, wxRibbonControl);
 
-class wxRibbonGalleryItem
+class wxRibbonGalleryItem : public wxSharedClientDataContainer
```

Comment:
Not sure if this is really needed. I guess I don't have any real objections to it neither, but as long as the items are not copied/cloned, this just doesn't seem necessary. But we can keep it if you see value in it.